### ISSUE TITLE
Beautify breaks fancy attribute selectors like ~=

### ISSUE THREAD

**Issue by ivanreese** [jakubpawlowicz/clean-css#953]
### Input CSS
I love these fancy [attribute selectors](http://mdn.io/attribute-selector)!

```css
cd-row > [col~="5th"] {
  order: 5;
}
```

### Actual output CSS
Oh.. pumpkins! Note the spaces around the `~`. In decent browsers, well, they'll just roll with the punches and ignore the spaces. But in IE11, it's a syntax error, so the entire rule is ignored.

```css
cd-row > [col ~ ="5th"] {
  order: 5
}
```

### Expected output CSS
Mmm that's the stuff.

```css
cd-row > [col~="5th"] {
  order: 5
}
```

### Configuration options
The `-O` level doesn't have an effect. This issue seems to entirely be the fault of the beautification. If the beautification is turned off, the output CSS is `cd-row>[col~="5th"]{order:5}`, which is correct.

```bash
cleancss -o clean.css styles.css -O 0 --format beautify
```

### Environment

* clean-css version - `npm ls clean-css`: 4.1.5
* node.js version - `node -v`: v8.1.0
* operating system: macOS 10.12.5

**Comment by jakubpawlowicz** [comment]
Yup, this is wrong. Fix is coming!

**Comment by jakubpawlowicz** [comment]
Fixed in 4.1.6.
